docs/faq.md

```diff
@@ -291,3 +291,16 @@ Currently, AKO beyond two has not been tested. Hence we don't claim the support.
 #### What happens during a Split brain scenario in HA?
 
 AKO detects this during the periodic refresh of the lease lock object and makes itself a passive AKO.
+
+#### With AKO 1.10.3, field `networkUUID` has been added as part of `vipNetworkList` and `nodeNetworkList`. Does it impact existing AviInfra objects?
+
+When customer upgrade from older AKO version to AKO 1.10.3 or later, customer has to always upgrade CRD schemas as part of upgrade process.
+But there is no impact on existing AviInfraSetting objects and user doesn't require to do changes to existing objects. AKO accepts either `networkName` or `networkUUID` as part of these fields.  If there are duplicates network at NSX Advance Loadbalancer side, AKO recommends the use of `networkUUID` while creating new AviInfrasetting objects.
+
+#### How can I find out `networkUUID` for the `networkName`?
+
+One of the way to figure out `networkUUID` for given `networkName` is as follows:
```

Comment:
Should we list down all the ways or do we have to reword this if this is the only way?